package.json

```diff
@@ -91,8 +92,9 @@
   },
   "scripts": {
     "build": "rm -rf dist && NODE_ENV=production babel src --out-dir dist --copy-files --extensions '.js,.jsx,.ts,.tsx'; yarn build-declaration",
+    "build-local": "NODE_ENV=production babel src --out-dir dist --copy-files --extensions '.js,.jsx,.tsx'",
     "build-declaration": "tsc",
-    "build-watch": "yarn build --watch",
+    "build-watch": "watch 'yarn run build-local' ./src",
```

Comment:
Why using `watch` here? Isn't `--watch` flag for babel enough?

It seems that before TS was introduced `yarn build --watch` was working, because it was passing `--watch` to babel. Now `tsc` is last step of build, so `--watch` is being passed to TS instead.
If the babel build is extracted to `build-local`, maybe `yarn build-local --watch` would work?

Reply:
But I'm not really sure how watch should work here if we have babel and TS steps in build.
@squidsoup Any thoughts on that Kit?

Reply:
Good catch.. don't need the watch package afterall